.circleci/config.yml

```diff
@@ -172,70 +126,43 @@ jobs:
 workflows:
   version: 2
 
-  # The main workflow responsible for triggering all other workflows
-  # in which changes are detected.
-  ci:
-    when: << pipeline.parameters.trigger >>
-    jobs:
-      - trigger-workflows
-
-  # Trigger tag build when tag and not already building a specific package workflow
-  tag:
-    when:
-      and:
-        - << pipeline.git.tag >>
-        - not: << pipeline.parameters.wpe-headless-plugin >>
-    jobs:
-      - trigger-tag:
-          filters:
-            tags:
-              only: /^plugin\/wpe-headless\/\S+/
-
   # Workflows defined for each package.
-  # By default circleci doesn't trigger on a tag event. Add the filter for tags so that all jobs in the workflow will run also when a tag is present.
+  # tag example for deploying an update for wpe-headless: plugin/wpe-headless/1.0.0
   plugin-wpe-headless:
-    when: << pipeline.parameters.wpe-headless-plugin >>
     jobs:
-      - checkout:
-          filters:
-            tags:
-              only: /.*/
+      - checkout
       - plugin-wpe-headless-checkout:
           requires:
             - checkout
-          filters:
-            tags:
-              only: /.*/
       - plugin-wpe-headless-composer:
           requires:
             - plugin-wpe-headless-checkout
-          filters:
-            tags:
-              only: /.*/
       - plugin-wpe-headless-bundle-zip:
           requires:
             - plugin-wpe-headless-checkout
           filters:
+            branches:
+              ignore: /.*/
             tags:
-              only: /.*/
```

Comment:
Why are you changing this behavior?

Reply:
Since this isn't run using the trigger workflow anymore, my thought is that it should be triggered at the same time as `plugin-wpe-headless-deploy`

Reply:
Ok, see my comment below about deploying a zip file to staging api when main or canary.